### ISSUE TITLE
container around the image

### ISSUE THREAD

**Issue by monsieur42** [sachinchoolur/lightGallery#80]
It would be fine if there was a container around the image which follows the size of the image and the navigation, the close button, the thumbnails and the custom html would be rendered there because in this way the customization of the layout would be easier. What do you think?

**Comment by sachinchoolur** [comment]
Great suggestion.
Many people requested me the same but I thought css only layout (without using javascript) is better.
Anyway this is a much needed addition and will be included in the next version..

Thanks

**Comment by monsieur42** [comment]
Great! Actually I do use javascript to align the navigation buttons and other stuffs to the image becouse I couldn't find any better way. If you have any better solution to do this before the next version, please let me know. 
It's the best image gallery plugin I found anyway. Thank you for all! :)

**Comment by monsieur42** [comment]
Sorry but I found an other issue. When I put a <br> tag after a <li> in the gallery it generates a blank page. How can I fix it?

**Comment by sachinchoolur** [comment]
It would be better if you could provide a demo for your setup.

**Comment by monsieur42** [comment]
Ok! I'm really noobish! :D The html tags doesn't shown in my previous post... lol
So, here it is:
http://www.pomsarphotography.hu/index.php/hu/?option=com_content&view=article&id=135
After the third thumbnail's "li" there is a "br" tag and if you open the images in the gallery view after the third image there will be a blank page.

**Comment by sachinchoolur** [comment]
Hi,
First of all the direct children of a ul element must all be li elements. 
However you can use selector property for fixing this issue.
``` html
 <ul id="lightGallery">
    <li class="portfolio_li"></li>
    <li class="portfolio_li"></li>
    <br />
    <li class="portfolio_li"></li>
</ul>
```
``` javascript
$('#lightGallery').lightGallery({
    selector: '.portfolio_li'
})
```
http://jsfiddle.net/sachin377/p1aw7utk/5/

**Comment by monsieur42** [comment]
Ok! Thanks. :)

**Comment by sachinchoolur** [comment]
Hi,
New release was not a major one. I will include this feature in lightGallery version 1.2. It is already under development. If you want i will give you a demo within two days.

**Comment by monsieur42** [comment]
Oh! That would be cool! Thanks a lot! :)

**Comment by smartcorestudio** [comment]
+1 My usecase: I'm using one gallery for images, located in different parts of the page, not in one list

**Comment by sachinchoolur** [comment]
Hi,
Please have a look at this link http://sachinchoolur.github.io/
This is just for a demo not even 50% completed. I will try to release new version as soon as possible.

**Comment by monsieur42** [comment]
Awesome! It's just what I needed! :)

**Comment by MasTArrrRR** [comment]
Hey I first of all want to say I love your gallery. Great work. Secondly I was wondering how I would go about centering all the thumbs that display prior to clicking on them for the enlargement of the gallery. Meaning when the thumbs are just there, they're all aligned to the left I would like to know how to center them so they look nicer on the website. Secondly, is there a way to make a nice thin border around every thumb that will automatically go there as I add more pictures? I hope to hear from you soon. You're amazing and keep up the great work Boss.

Sincerely Joe
yydesigns@gmail.com.

**Comment by sachinchoolur** [comment]
If you want thumbnails animation while moving the slide you have to align it to left. But still you can set `currentPagerPosition: 'middle'` to center the active thumbnail.
If you are not using `animateThumb: true` you can align it however you want using css.
If you want border around thumb you can add it via css..

**Comment by sachinchoolur** [comment]
Hi guys,
I have released lightgallery version 1.2 with this feature..
Please check.
